### ISSUE TITLE
Voice control not working mac

### ISSUE THREAD

**Issue by mikey0000** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2171]
**OS:**
Mac OSX Sierra
**Issue Descriptions:**
experimental voice controls fail
**Steps to Reproduce:**
enable voice control
"hey music" etc doesn't do anything

I've also tested the google speech api example which also rendered nothing. I am using a headset with mic.

Looks like it fails then turns off, am happy to test myself and do pr, just need to validate that it is indeed a bug or my voice just isn't US sounding enough being a Kiwi

also looks like we should be setting the dialect too `recognition.lang = select_dialect.value;`

**Comment by MarshallOfSound** [comment]
@mikey0000 The language should be pulled in from your system locale.

(I am a Kiwi and it works for me 👍 )

However the voice API is severely rate limited due to Electron's API key so it only works during certain random hours 😆

**Comment by mikey0000** [comment]
Ah one of those issues.

voice API is severely rate limited due to Electron's API -- which API is 
this? can we allow personal API keys?

**Comment by MarshallOfSound** [comment]
@mikey0000 I believe the API key is baked in during the Electron build process.  But you can try launching the application from the command line with the `GOOGLE_API_KEY` environment variable set.

**Comment by gosteev** [comment]
@MarshallOfSound so i should have something like these?

![image](https://cloud.githubusercontent.com/assets/6203574/25909144/def52f6a-35b4-11e7-99bc-e0f5b34b1d8a.png)

and why should i launch app from command line?

**Comment by MarshallOfSound** [comment]
@gosteev Typically environment variables aren't picked up on macOS unless the app is launched from the command line.  Safer to suggest launching from the command line so I don't get "but it doesn't work on macOS" later down the line 👍 

But yeah, that should work.

**Comment by gosteev** [comment]
v4.3.0, win10 x64, no results. 
is that possible to create some type of 'sandbox' to test this feature?

**Comment by andrewdavidmackenzie** [comment]
I activated the feature for the first time (I tried a restart also) and "Hey music, etc" doesn't seem to do anything.


I can't see any other explanation of how to enable or use this feature in the docs.....and so assume no additional setup or Mac OS enabling is needed. Although I'd be slightly surprised if it's listening all the time without having to enable somewhere else?

Mac OS Sierra
10.12.6

Chrome
Version 59.0.3071.115 (Official Build) (64-bit)
